src/styles/custom.scss

```diff
@@ -29,6 +29,15 @@ pre,
   text-align: center;
 }
 
+.sticky-menu {
+  @media screen and (min-width: 768px) {
```

Comment:
This will need to be min-width: 769px to match Bulma's column breakpoint.

Other than that, good to go 👍

Reply:
I've updated it :+1: